package.json

```diff
@@ -71,7 +71,7 @@
     "jsdom": "11.2.0",
     "jsdom-global": "3.0.2",
     "lerna": "2.8.0",
-    "node-sass": "4.5.3",
+    "node-sass": "4.12.0",
```

Comment:
any reason why upgrade `node-sass`?

Reply:
I forget exactly, but I think I couldn't get the dev environment to build on Ubuntu 18.04